p2p/Cargo.toml

```diff
@@ -39,7 +39,8 @@ prost-amino = { version = "0.6", optional = true }
 prost-amino-derive = { version = "0.6", optional = true }
 
 [dev-dependencies]
-pipe = { version = "0", features = ["bidirectional"] }
+crossbeam-channel = "^0.5.0"
```

Comment:
Can we converge on one channel implementation?

Reply:
while it should be possible to rewrite `pipe` impl to use `flume` instead of `crossbeam-channel`, `flume` does not support (nor does it plan to) broadcast channels https://github.com/zesterer/flume/issues/68#issuecomment-761032975. I'm using broadcast channels in https://github.com/informalsystems/tendermint-rs/pull/825/files#diff-6adb8bb700e22bde6a4c4e334b9e1a36fafae1026f988f9d52836690ed97cf68R88 to propagate data read from socket to virtual streams.

Reply:
Broadcast semantics shouldn't be too hard to build, given that we will hold onto the set of all virtual streams.